### ISSUE TITLE
Concerns on the `AdversarialTrainingClassifier`

### ISSUE THREAD

**Issue by xuyxu** [xuyxu/Ensemble-Pytorch#38]
Hi @zzzzwj, I cannot reproduce the experiment results reported in the paper : Simple and Scalable Predictive Uncertainty
Estimation using Deep Ensembles.

It is true that the training loss is smaller than that when training a single base estimator, but the performance after ensembling is not much better than using a single estimator.

We may need to dig in the `AdversarialTrainingClassifier` and check where goes wrong.

**Comment by zzzzwj** [comment]
Sorry, I don't know either. We may try to train our ensemble model in a larger dataset. The cifar10 is probably saturated to some extent.

**Comment by xuyxu** [comment]
Is that true that if `AdversarialTrainingClassifier` fails on CIFAR-10, adding adversarial samples on a harder dataset, say, CIFAR-100, will degrade its performance further?

**Comment by zzzzwj** [comment]
It's just my guess and needs to be validated.
